packages/react-app/pages/index.jsx

```diff
@@ -70,7 +75,9 @@ export async function getStaticProps() {
         buildsIncrementMonth: stats?.buildsIncrementMonth,
         streamedEthIncrementMonth: stats?.streamedEthIncrementMonth,
       },
+      events,
     },
+    // ToDo. Maybe a 15 min refresh? or load events in the frontend?
```

Comment:
NO need to fix in this PR, but we have to think if:

1. We decrease this number to 15/30 min
2. We leave it as it's, but we fetch the activity from the frontend.

Not sure what's best (it depends on what we want to optimize)

Reply:
> Not sure what's best (it depends on what we want to optimize) 

Yeah, btw 15 / 30 min doesn't seem too low ? like if we keep it 15 / 30 min maybe its better we fetch it on frontend itself so we always show latest data to user ? Assuming that app.bg.com is visited less 

Maybe we decrease it from 6hr to 3hr ? 

Lol just thinking out loud, and really don't good idea / context how frequent bg events are emitted and totally agree that what we want to optimize !

Reply:
> Yeah, btw 15 / 30 min doesn't seem too low

Might be! 

If we don't cache anything, we are going to make a bunch of server/database requests on every visit to the homepage. I was thinking that even if we have 30 min as a caching window, we could save some.

I was trying to have a small number so the activity shows up to date.

But I think you are right, maybe we can start with 2/3 hours, and see.

We can always cache everything 6 hours, and just have the activity fresh, fetching from the frontend

Thanks Shiv!